1001-2000/LIT1936Mashaf.xml

```diff
@@ -1,12 +1,24 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1936Mashaf">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Mäṣḥafä ḥǝmamatihu wäzena sǝqlätu wäqǝtlätu läʾǝgziʾǝna</title>
-            <author>Mäbaʾa Ṣǝyon</author>
+            <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ሕማማቲሁ፡ ወዜና፡ ስቅለቱ፡ ወቅትለቱ፡ ለእግዚእነ፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#t1">Maṣḥafa ḥǝmāmātihu wa-zenā sǝqlatu wa-qǝtlatu la-ʾƎgziʾǝna</title>
+            <title xml:lang="en" corresp="#t1">Book of Suffering and Narration of the Crucifixion and Killing of our Lord</title>
+
+            <title xml:lang="en" xml:id="t2">The Book of the Passion of Our Lord</title>
+
+
+            <title xml:lang="gez" xml:id="t3">ተዝንካረ፡ መድኅን፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#t3">Tazkāra Madḫǝn</title>
+            <title xml:lang="en" corresp="#t3">Memorial of the Passion and Death of Christ</title>
```

Comment:
Why "Memorial (= "Denkmal"?) of the passion and death"? It is a liberal translation of the Geez title. Probably, in order to understand why the title isተዝንካረ፡ መድኅን፡ and how exactly it is meant, one need to read deeper into the text, but for the time being we can just translate it literally as "Commemoration of the Saviour".